Mathlib/Analysis/SpecificLimits/Normed.lean

```diff
@@ -640,6 +640,69 @@ theorem Antitone.tendsto_alternating_series_of_tendsto_zero (hfa : Antitone f)
 
 end
 
+section
+
+/-! ### Partial sum bounds on alternating convergent series -/
+
+
+variable {E : Type*} [OrderedRing E] [TopologicalSpace E] [OrderClosedTopology E]
+  {l : E} {f : ℕ → E} (hfl : Tendsto (fun n ↦ ∑ i in range n, (-1) ^ i * f i) atTop (𝓝 l))
```

Comment:
Prop assumptions like `hfl` should be mentioned in the theorems, not in the `variable` line: the reason is that otherwise the file is very hard to understand for the readers, as they don't know which assumptions are used.

Reply:
Done.